source/_includes/asides/docs_navigation.html

```diff
@@ -46,6 +46,10 @@ <h1 class="title delta">Topics</h1>
           <li>{% active_link /docs/assist/apple/ Assist for Apple devices %}</li>
           <li>{% active_link /docs/assist/builtin_sentences/ Built-in sentences %}</li>
           <li>{% active_link /docs/assist/custom_sentences/ Custom sentences %}</li>
+          <li>{% active_link /projects/private-voice-assistant/voice_remote_esp32/ Setting up a $13 voice remote to control Home Assistant %}</li>
+          <li>{% active_link /projects/private-voice-assistant/voice_remote_local_device/ Setting up a voice remote that processes locally %}</li>
```

Comment:
These 2 are projects but are linked in the docs navigation, should they just be under docs?

Also, the analog phone is the World's Most Private Voice Assistant. Currently it has 2 folders `private-voice-assistant/voice_remote_local_device/`. Let's change this to `/projects/worlds-most-private-voice-assistant`

The ESP tutorial is the "$13 voice remote for Home Assistant". It currently also has 2 folders. Let's change this to `/projects/13-usd-voice-remote`